cypress/e2e/specialFeatures/atiAnalytics/index.cy.js

```diff
@@ -246,6 +246,7 @@ const canonicalTestSuites = [
     pageIdentifier: 'marathi.topics.c1wmk63rjkvt.page',
     applicationType: 'responsive',
     contentType: 'index-category',
+    componentTrackingContentType: 'topic-page',
```

Comment:
Was this an oversight, or was it always designed this way. It's a real shame that the 2 are not the same, and that we have to do this, but 🤷‍♀️

Reply:
I'll tag you in the slack conversation about this. It is for continuity in the data reported to Piano.

Reply:
I guessed it would be, thanks!